src/venue/core.cljs

```diff
@@ -120,6 +120,13 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(defn- init-services
+  []
+  (doseq [service (:services @state)]
+    (let [handler ((val service))]
+      (when (satisfies? IInitialise handler)
```

Comment:
The preference here is to remove the `satisfies?` check but extend the protocol to java.lang.Object and provide a no-op initialise fn. You can also extend the protocol to nil, which can be useful, but consider if it's better to throw the NPE in that case (it depends).